src/Features/CSharp/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -59,6 +59,11 @@ internal partial class CSharpIntroduceVariableService
                 return Task.FromResult(IntroduceLocalDeclarationIntoLambda(
                     document, expression, newLocalName, declarationStatement, parentLambda, allOccurrences, cancellationToken));
             }
+            else if (IsInExpressionBodiedMember(expression))
```

Comment:
Both cases seem to work, though the one with anonymous method exposed a formatting bug after the refactoring completes. It could be the refactoring itself with how it manipulates trivia or a bug in indentation service.